sdk/go/common/resource/config/testdata/repr/21.yaml

```diff
@@ -1 +1 @@
-my:test.Key: testValue
```

Comment:
test.Key wasn't actually valid and didn't work with Get. See how the old 21.expected.yaml had "" for the value and string